arcsi/templates/data/uploaded_episodes.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block header %}
+<h2>{% block title %}Uploaded episodes{% endblock %}</h2>
+<h5>This query gives back the number of uploaded episodes in the selected timeframe.</h5>
+<h5>Both date are inclusive!</h5>

```

Comment:
Both end dates are included!

Reply:
Yepp, it's more clear that way, fixed it!

Reply:
Hmm, it's still the old text (Both date are inclusive!). It's only grammer, so let's not block the PR and go with it.